### ISSUE TITLE
IC2 biogas recipe

### ISSUE THREAD

**Issue by nictrace** [Blood-Asp/GT5-Unofficial#1291]
What is IC2 biogas? Maybe, dirty methane? Can you add separation recipe for centrifuge biogas to methane, CO2 and maybe H2S?

**Comment by leagris** [comment]
@nictrace Do you have something in mind needing pure CH₄ methane, instead of renewable Biogas, or recipes needing CO₂ or H₂S?

**Comment by nictrace** [comment]
Currently, we have recipe for methane -> biogas, but how about law of conservation of mass?)
Methane, of course, can produce more energy in gas turbine.

**Comment by Antillar** [comment]
I see what you want. PTFE and Silicone Rubber from food. No freaking way!

**Comment by nictrace** [comment]
@Antillar, but first syntetic rubbers were made from potato really!

**Comment by Antillar** [comment]
@nictrace according to Wikipedia, first synthetic rubber was made from 2,3-dimethylbutadiene.

**Comment by nictrace** [comment]
yes, and butadiene was made from ethanol according to Lebedev method

**Comment by PrincessOfEvil** [comment]
Why is rubber from food bad, from a balance standpoint? Synthetic rubber made from potatoes won't be of any use, though, pure polybutadiene is very soft.

**Comment by Antillar** [comment]
Ok, I see my mistake here. Polystyrene WAS made from "food". I admit it.

From the point of the balance:
Let us imagine that the original idea is implied. The methane can be made from IC2 biogas. 
Under certain relatively easily achievable conditions the player can automate the biogas production with the usage of saplings and/or leaf blocks, which are a common byproduct of a simple tree farm. Let us skip the stage at which this all converts into biogas. We all know this stage, so it is not worth going into details here.
Basically, at this point, the player can create an infinite biogas supply. This originates as a concept of one of the autonomous power sources, and, if tweaked precisely, can produce additional biogas at a rapid rate over the amount required to power itself. 
At the following stage the biogas can be converted into methane, which then makes its way into PTFE and Silicone Rubber. I aspecially note that no hydrocarbons other than methane are required to create these two valuable products. And another detail: this all goes at highest at MV tier, and more likely it can be achieved at LV tier.

What do we have in total?
* Infinite automatic autonomous supply of methane.
* Conditionally infinite supply of Silicone Rubber.
* Ability to unlock PTFE as soon as Fluorine is obtained, which is MV tier, or probably LV tier.
* Styrene-Butadiene Rubber is never going to be produced by the said player, unless as an ingredient for LuV and ZPM conveyor belts, required in limited amount for Advanced Circuit Assembler V and VI respectively in order to craft wetware circuity. 
* Instant access to all high tier cables, provided that the player is able to smelt the metal they are made from.
* Instant access to LCR at MV tier.
* Polyethylene no longer required to produce other than for the cheap recipe of LV-IV machine hulls.

I am likely overcomplicating things, but this is exactly what I would have done if I had had this ability from the start.

**Comment by nictrace** [comment]
No problem! BC farm for rubber trees - infinite rubber, also we have direct recipes for pumpkin->methane and potato->methane

**Comment by Antillar** [comment]
Rubber is originally meant to be infinite. I never understood any opinions on making it something hard. It stays infinite and easily automatable for over 6 years, and let it be so.

As far as I am concerned, pumpkin farm and potato farm (so is unmentioned above carrot farm) cannot power itself, and even if paired with oak farm into an autonomously functioning duo, does not produce the materials fast enough to be even compared to oak farm. 25x25 square of oak saplings has over 1 tree growing every second (from my significant experience in operating these), this yields 3 saplings and 50 leaf blocks in average. I do note that there is an extremely wide variety of the ways to harvest the said trees, even with the usage of the Wither sitting in a minecart using only vanilla blocks and items. 

Add this to the fact that almost every player does automate an oak farm at some point for different reasons. And how many people do actually automate giant fields of vegetables? 

I am not saying that the concept is impossible at this point. I am saying (well, trying to) that the idea suggested here draws a bold straight line for most of the players to do this. Go on, screw your earlygame!

I see that my arguments are more of being subjective, and clearly shouldn't be taken by other people in this conversation without any doubt, so I think that the only possible way to solve this problem is to come for an agreement on somehow limiting the ability of the players to create setups described above.

Personally I suggest locking this ability behind HV tier. To be more precise, I suggest setting a voltage requirement of biogas->methane recipe to 480 EU. It seems rather reasonable for me that such of a shortcut should be opened after the player has at least made integrated processors. This definitely implies him/her to find, pump and process oil in order to obtain polyethylene to craft the said processors, so the shortcut for at least some hydrocarbons could be handy at this particular point and will definitely not ruin the experience for the said player.
